CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ All notable changes to this project will be documented in this file.
 - Added support for partial read for MWA correlator FITS files.
 - Added `antenna_names`, `time_range`, `lsts` and `lst_range` parameters to
 `UVFlag.select` to match UVData and UVCal select methods.
+- New convenience methods on `BeamInterface` to simplify the handling of analytic vs
+UVBeam objects.
```

Comment:
This should go at the top of this section, it should be reverse chronological order.